guiaColombia/settings.py

```diff
@@ -30,6 +30,7 @@
 ALLOWED_HOSTS = ['guia-col-django.herokuapp.com']
 
 # Application definition
+CORS_ORIGIN_ALLOW_ALL=True
```

Comment:
Solo por tenerlo en cuenta, esta bandera permite que cualquier dirección pueda acceder a los recursos que se expone en la aplicación, algo que en la vida real no debería permitirse...

Configuration
Configure the middleware’s behaviour in your Django settings. You must add the hosts that are allowed to do cross-site requests to CORS_ORIGIN_WHITELIST, or set CORS_ORIGIN_ALLOW_ALL to True to allow all hosts.

CORS_ORIGIN_ALLOW_ALL
If True, the whitelist will not be used and all origins will be accepted. Defaults to False.

CORS_ORIGIN_WHITELIST
A list of origins that are authorized to make cross-site HTTP requests. Defaults to [].